templates/compose-main.tmpl

```diff
@@ -108,6 +108,19 @@ services:
         restart: on-failure
         command: server
 
+    jaeger:
+        image: jaegertracing/all-in-one:1.13
+        networks:
+            - {{{get . "DOCKER_NETWORK_NAME"}}}
+        ports:
+            - "5775:5775/udp"
```

Comment:
We shall not open these ports

Reply:
I removed the unnecessary ports. Only 2 left: 
- 6831/udp - this one is used by the clients to send the tracing packages
- 16686 - this is for the Jaeger UI

16686 should be accessible from the outside for obvious reasons.
6831/udp has to be exposed because otherwise services in local dev mode wouldn't be able to send packages.